src/components/Blog/Blog.jsx

```diff
@@ -44,14 +45,32 @@ const Blog = () => {
           return blog;
         });
       });
-      alert("You have liked the blog");
+      toast("You have liked the blog", {
```

Comment:
toast.success would be more appropriate here

Reply:
done